src/site/content/en/blog/optimize-vitals-lighthouse/index.md

```diff
@@ -0,0 +1,365 @@
+---
+layout: post
+title: Optimizing Core Web Vitals using Lighthouse
+subhead: Finding opportunities to improve user-experience with Chrome's web tooling.
+authors:
+  - addyosmani
+description: |
+  Today, we will cover new tooling features in Lighthouse, PageSpeed and DevTools to help identify how your site can improve on the Core Web Vitals.
+date: 2021-05-10
+hero: image/1L2RBhCLSnXjCnSlevaDjy3vba73/6GPqQDYxZnVq8qF6DJ02.jpeg
+alt: "A Lighthouse illuminating the sea"
+tags:
+  - blog
+  - fast
+  - performance
+  - web-vitals
+---
+
+Today, we'll cover new tooling features in Lighthouse, PageSpeed and DevTools to help identify
+how your site can improve on the Core Web Vitals.
+
+As a refresher on the tools, [Lighthouse](https://github.com/GoogleChrome/lighthouse) is an
+open-source, automated tool for improving the quality of web pages. You can find it in the [Chrome
+DevTools](https://developers.google.com/web/tools/chrome-devtools) suite of debugging tools and run
+it against any web page, public or requiring authentication. You can also find Lighthouse in
+[PageSpeed
+Insights](https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fstore.google.com),
+[CI](https://github.com/GoogleChrome/lighthouse-ci) and
+[WebPageTest](https://www.webpagetest.org/easy). 
+
+Lighthouse 7.x includes new features like element screenshots, for easier visual inspection of
+parts of your UI impacting user-experience metrics (e.g. what nodes are contributing to layout
+shift).
+
+
+<figure class="w-figure">
+<video muted autoplay loop>      
+<source type="video/mp4" src="https://storage.googleapis.com/web-dev-uploads/video/1L2RBhCLSnXjCnSlevaDjy3vba73/3G0x4Z1dmOcsusG7j1LE.mp4" type="video/mp4" width="1920" height="1080">    
+</video>
+</figure>
+
+We've also shipped support for element screenshots on PageSpeed Insights, enabling a way to more
+easily spot issues for one-off performance runs of pages.
+
+<figure class="w-figure">
+  {% Img src="image/1L2RBhCLSnXjCnSlevaDjy3vba73/mfkWFzyfO9XlJLYS80DE.png", alt="Element Screenshots highlighting the DOM node contributing to layout shift in the page", width="800", height="483" %}
+</figure>
+
+
+## Measure Core Web Vitals
+
+Lighthouse can
+[synthetically](/vitals-measurement-getting-started/#measuring-web-vitals-using-lab-data) measure
+the [Core Web Vitals metrics](/vitals/) including [Largest Contentful Paint](/lcp/), [Cumulative
+Layout Shift](/cls/) and [Total Blocking Time](/tbt/) (a lab proxy for [First Input Delay](/fid/)).
+These metrics reflect loading, layout stability, and interaction readiness. Other metrics such as
+[First Contentful Paint](/first-contentful-paint/) highlighted in the [future of
+CWV](https://developer.chrome.com/devsummit/sessions/future-of-core-web-vitals/) are there too.
```

Comment:
Can you spell out the acronym here. I don't think we ever use the abbreviation CWV externally.

Reply:
Certainly. I'll spell this out in a follow-up.